mbuild/tests/test_cassandramcf.py

```diff
@@ -63,6 +63,40 @@ def test_no_dihedrals(self,methane):
 
         assert mcf_data[dihedral_section_start+1][0] == '0'
 
+    def test_no_14(self,methane):
+        methane.save(filename='methane-opls.mcf', forcefield_name='oplsaa',
+                angle_style='harmonic', dihedral_style='none')
+
+        mcf_data = []
+        with open('methane-opls.mcf') as f:
+            for line in f:
+                mcf_data.append(line.strip().split())
+
+        for idx,line in enumerate(mcf_data):
+            if len(line) > 1:
+                if line[1] == 'Intra_Scaling':
+                    intrascaling_section_start = idx
+
+        assert mcf_data[intrascaling_section_start+1][2] == '0.0000'
```

Comment:
Are these decimal places in the MCF file always 4? If so, that seems fine, if not, might be wise to use something like an `np.allclose` function.

Reply:
It is always written to four decimal places.